### PR TITLE
feat: introduce bigtable-hbase-1x-compat

### PR DESCRIPTION
**Pull request by igorbernstein2**
This repackages hbase-shaded-client for bigtable-hbase users that still depend on HBase 1 apis. A follow up PR will update the integration tests to test against both hbase-shaded-client, hbase-client and this new artifact

Change-Id: I6fbe807cdb747c0b2354bd95dd4fbb990c40337f

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-bigtable-hbase/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

File: bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>bigtable-hbase-1.x-parent</artifactId>
+        <groupId>com.google.cloud.bigtable</groupId>
+        <version>2.7.3-SNAPSHOT</version> <!-- {x-version-update:bigtable-client-parent:current} -->
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>hbase-client-1x-compat</artifactId>
```
Comment (kolea2) [inline bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml:15]: nit - rename to `1.x` here and in the folder name

File: bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+            <dependency>
+                <groupId>commons-codec</groupId>
+                <artifactId>commons-codec</artifactId>
+                <version>1.15</version>
+            </dependency>
+            <!-- resolve conflict between hbase-common and hadoop-common -->
+            <dependency>
+                <groupId>commons-io</groupId>
+                <artifactId>commons-io</artifactId>
+                <version>2.11.0</version>
```
Comment (kolea2) [inline bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml:54]: please add to renovate if you do not want to bump this version

File: bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml [hunk 3]
```diff
@@ -0,0 +1,527 @@
...
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-log4j12</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>com.github.stephenc.findbugs</groupId>
+            <artifactId>findbugs-annotations</artifactId>
+            <version>1.3.9-1</version>
```
Comment (kolea2) [inline bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml:133]: please add to renovate here and below if needed

File: bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml [hunk 4]
```diff
@@ -0,0 +1,527 @@
...
+                <version>2.11.0</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.hbase</groupId>
+            <artifactId>hbase-client</artifactId>
+            <version>1.7.2</version>
```
Comment (kolea2) [inline bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml:63]: replace with variable

**Review by kolea2 — CHANGES_REQUESTED** [review]

File: bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml (hunk 2 above)
Comment (igorbernstein2) [reply bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml]: All of the versions here are fair game to update

File: bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml (hunk 4 above)
Comment (igorbernstein2) [reply bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml]: done

File: bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml (hunk 1 above)
Comment (igorbernstein2) [reply bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml]: done

**Pull request merged**